### PR TITLE
Buyupdate

### PR DESCRIPTION
**Pull request by jospoortvliet**
Updating the 'buy' page to also inquire after needs for branding, outlook and WebRTC.
Also improved the looks a little, making 2 columns for the list of needs & making the button less ugly.

### CODE REVIEW

**Comment by jospoortvliet** [comment]
buy nextcloud 26-9-2017 22-01-56-fs8

File: page-buy.php [hunk 1]
```diff
@@ -47,43 +47,68 @@
...
+                            <option value="yes"><?php echo $l->t('Yes');?></option>
+                            <option value="no"><?php echo $l->t('No');?></option>
+                        </select></label></p>
+                        <p><label for="outlook"><?php echo $l->t('Would you be interested in our Secure Sharing add-in for Outlook?');?><br>
+                        <select name="outlook">
+                            <option value="unsure"><?php echo $l->t('Not sure');?></option>
+                            <option value="yes"><?php echo $l->t('Yes');?></option>
+                            <option value="no"><?php echo $l->t('No');?></option>
+                        </select></label></p>
+                        <p><label for="branding"><?php echo $l->t('Would you be need branded clients?');?><br>
```
Comment (MorrisJobke) [inline page-buy.php]: Isn't `Would you need branded clients?` a better wording?

Comment (MariusBluem) [reply page-buy.php]: 👍

Comment (jospoortvliet) [reply page-buy.php]: Fixed. Anything else?

**Comment by nickvergessen** [comment]
Reads good but didnt checkout to see if it parses

**Pull request merged**